tokio/src/net/udp.rs

```diff
@@ -443,6 +443,39 @@ impl UdpSocket {
         Ok(())
     }
 
+    /// Polls for write readiness.
+    ///
+    /// If the udp stream is not currently ready for writing, this method will
+    /// store a clone of the `Waker` from the provided `Context`. When the udp
+    /// stream becomes ready for writing, `Waker::wake` will be called on the
+    /// waker.
+    ///
+    /// Note that on multiple calls to `poll_write_ready` or `poll_write`, only
```

Comment:
There is no `poll_write` method on `UdpSocket`. This should instead mention `poll_send` and `poll_send_to`.